### PR TITLE
fix: cursor now changes to pointer when hovering

### LINKED ISSUE #890: The link area on the policy editor right-hand panel's individual parameter scroll component doesn't change to pointer
Very minor edit; e.g., when hovering over the blue "Enable" to click on that policy editor screen, the cursor doesn't become a pointer

Screen Shot 2023-11-23 at 5 23 17 PM

### PR DESCRIPTION
**Pull request by dotslashbit**
## Description

fixes #890 

## Changes

updated the cursor now change to pointer when hover over the enable

## Screenshots

https://github.com/PolicyEngine/policyengine-app/assets/46224156/4e5db4e1-e2fd-4e7d-8a91-10a2b6f9022c

### CODE REVIEW

File: src/pages/policy/PolicyRightSidebar.jsx
Comment (anth-volk) [inline src/pages/policy/PolicyRightSidebar.jsx:1]: Unfortunately, this change will work only for boolean parameter changes, but not for value changes (e.g., https://policyengine.org/us/policy?reform=43695&focus=policyOutput.policyBreakdown&region=us&timePeriod=2024&baseline=2). Could you also add the styling on post-change line 162?

**Review by anth-volk — CHANGES_REQUESTED** [review]
Thanks so much, @dotslashbit. I did leave one change request, but otherwise your efforts are much appreciated.

**Comment by dotslashbit** [comment]
@anth-volk  I have added the styling, please check and let me know if there's any changes needed

**Review by anth-volk — APPROVED** [review]
Thanks @dotslashbit, this looks perfect. Merging.

**Pull request merged**